src/backend/joanie/tests/core/test_signals.py

```diff
@@ -445,19 +447,22 @@ def test_signals_on_change_course_product_relation_set(self, mock_sync):
     @mock.patch.object(webhooks, "synchronize_course_runs")
     def test_signals_on_change_course_product_relation_create(self, mock_sync):
         """Product synchronization should be triggered when a product is created for a course."""
-        course_run = factories.CourseRunFactory()
+        course_run = factories.CourseRunFactory(state=CourseState.ONGOING_OPEN)
         product = factories.ProductFactory(target_courses=[course_run.course])
         course = factories.CourseFactory(products=[product])
         mock_sync.reset_mock()
 
         product_types = [
-            t for t, _name in enums.PRODUCT_TYPE_CHOICES if t != "certificate"
+            t
+            for t, _name in enums.PRODUCT_TYPE_CHOICES
+            if t != enums.PRODUCT_TYPE_CERTIFICATE
```

Comment:
since we take 3 lines, can be make it more explicit what `t` stands for ?
```suggestion
            product_type
            for product_type, _name in enums.PRODUCT_TYPE_CHOICES
            if product_type != enums.PRODUCT_TYPE_CERTIFICATE
```